apps/chat-e2e/src/assertions/downloadAssertion.ts

```diff
@@ -9,4 +11,14 @@ export class DownloadAssertion {
     expect(downloadedData.path).toBeTruthy();
     expect(downloadedData.path).toMatch(new RegExp(`${expectedExtension}$`));
   }
+
+  public async assertFileIsDownloaded(downloadedData: UploadDownloadData) {
+    const downloadedFiles = FileUtil.getExportedFiles();
+    expect
+      .soft(
+        downloadedFiles?.find((f) => f.includes(downloadedData.path)),
```

Comment:
shouldn't we check that the size of the file is > 0?

Reply:
added undefined value check